docs/providers.md

```diff
@@ -1,23 +1 @@
 # Providers and supported objects
-
-## s3
-- object
-- bucket
-
-## outscale_oapi
-- vm
-- load_balancer
-- nat_service
-- security_group
-- public_ip
-- volume
-- keypair
-- route_table
-- internet_service
-- subnet
-- net
-- image
-- snapshot
-- vpn_connection
-- virtual_gateway
-- nic
```

Comment:
It seems that `log.Printf` writes by default into the stderr that is why the script `providers.sh` gives no result.